README.md

```diff
@@ -27,6 +27,20 @@ cd $GOPATH/github.com/src-d/code-annotation
 make serve
 ```
 
+### Restrict access
+
+It's possible to restrict access to the tool or `reviewer` role by setting environmental variables:
+
+* `OAUTH_RESTRICT_ACCESS`
+* `OAUTH_RESTRICT_REVIEWER_ACCESS`
+
+As a value it accepts github organization name or github team id. Example:
+
+```bash
+OAUTH_RESTRICT_ACCESS=org:my-organization
+OAUTH_RESTRICT_REVIEWER_ACCESS=team:123456
+```
+
```

Comment:
May I suggest something more extended? The following text assumes that Reviewers are not a subgroup of the Workers. It is also worth noting that the current readme does not explain the two different roles.

### Access Control

It is possible to restrict access and choose each user's role by adding their GitHub accounts to a specific [organization](https://help.github.com/articles/collaborating-with-groups-in-organizations/) or [team](https://help.github.com/articles/organizing-members-into-teams/).

This is optional, if you don't set any restrictions all users with a valid GitHub account will be able to login as a Reviewer. You may also set a restriction only for Reviewer users, and leave open access to anyone as Workers.

To do so, set the following variables in you `.env` file:

* `OAUTH_RESTRICT_ACCESS`
* `OAUTH_RESTRICT_REVIEWER_ACCESS`

Both variables accept a string with either `org:<organization-name>` or `team:<team-id>`. For example:

```bash
OAUTH_RESTRICT_ACCESS=org:my-organization
OAUTH_RESTRICT_REVIEWER_ACCESS=team:123456
```